src/Data/ApplicationDbContext.cs

```diff
@@ -47,6 +47,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
             modelBuilder.Entity<Node>().HasIndex(x => x.PubKey).IsUnique();
             modelBuilder.Entity<Node>().HasIndex(x => x.Name).IsUnique();
+            modelBuilder.Entity<Wallet>().HasIndex(x => x.InternalWalletSubDerivationPath).IsUnique();
```

Comment:
I see a big issue here, the InternalWalletSubderivatonPath only holds 1 number, but it should be InternalWalletPath+SubPath, why? Because if there is a compromise and the internal wallet needs to change, the unique will make it impossible to start from 0 again.